fuzz.go

```diff
@@ -0,0 +1,49 @@
+// To fuzz-test this package:
+//
+//   $ go get github.com/dvyukov/go-fuzz/go-fuzz
+//   $ go get github.com/dvyukov/go-fuzz/go-fuzz-build
+//   $ go-fuzz-build github.com/dedis/protobuf
+//   $ go-fuzz -workdir=workdir -bin protobuf-fuzz.zip
```

Comment:
It is worthwhile putting the fuzzy tests on travis too?

Reply:
Please don't - having more flakyness is not what we need...

Reply:
No, they are long, in fact infinite running. So you run them while actively working on the parser only. I will teach a workshop on them sometime, it's really fascinating technology.